x/pairing/client/cli/tx_modify_provider.go

```diff
@@ -64,7 +63,7 @@ func CmdModifyProvider() *cobra.Command {
 		[chain-id] is the spec the provider wishes to modify the entry for
 		`,
 		Example: `lavad tx pairing modify-provider "ETH1" --gas-adjustment "1.5" --gas "auto" --gas-prices $GASPRICE --from <wallet>
-		lavad tx pairing modify-provider "ETH1" --endpoints "my-provider-africa.com:443,AF my-provider-europe.com:443,EU" --geolocation "AF,EU" --validator lava@valoper13w8ffww0akdyhgls2umvvudce3jxzw2s7fwcnk --from <wallet>`,
```

Comment:
sorry for the hustle, talked with omer and we dont want this change.
can you revert?

Reply:
LOL WHAT 😂

Reply:
Fixed in cee23f0de6fa9299c0b836f31a0552852fc93d0c